apps/web/.env.example

```
@@ -1,2 +1,10 @@
 NEXT_PUBLIC_DOCS_URL=https://your-docs.app.com
 NEXT_PUBLIC_WEB_URL=https://your-web.app.com
+BETTER_AUTH_SECRET=vg2vgnLmwcLu2UlDhFbLKuidprztXu1P
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The BETTER_AUTH_SECRET key should go before the NEXT_PUBLIC_DOCS_URL key

(UnorderedKey)